src/utils/shadow-css.ts

```diff
@@ -471,15 +473,32 @@ const scopeCssText = (
     cssText = scopeSelectors(cssText, scopeId, hostScopeId, slotScopeId, commentOriginalSelector);
   }
 
-  cssText = cssText.replace(/-shadowcsshost-no-combinator/g, `.${hostScopeId}`);
+  cssText = replaceShadowCssHost(cssText, hostScopeId);
```

Comment:
Tried to reorder this code so we didn't have to worry about doing this replacement in multiple places, but that caused other problems, so figured this was the solution that would cause the least headaches for now